Webtest.sh

```diff
@@ -44,4 +44,7 @@ docker exec fortio_server /usr/local/bin/fortio load -stdclient -qps 1 -t 2s -c
 docker exec fortio_server /usr/local/bin/fortio load -qps 1 -t 2s -c 2 http://www.google.com/
 # Do a grpcping
 docker exec fortio_server /usr/local/bin/fortio grpcping localhost
+# Test extra headers (-H) flag
+docker exec fortio_server /usr/local/bin/fortio load -H Foo:Bar -curl http://www.google.com
+docker exec fortio_server /usr/local/bin/fortio curl -H Foo:Bar -H Baz:Zab -curl http://www.google.com
```

Comment:
It will be a great e2e test in addition to the unit test I mentioned before

but you don't need -curl for curl subcommand - it's also redundant - do one with load and one with curl
in theory you'd want to see the headers are set but that's what the unit test is for